### PR TITLE
Update shinasystem.js

### PR DESCRIPTION
**Pull request by nukusinji**

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -719,6 +719,7 @@ const converters = {
                 {key: 'rmsVoltage', name: 'voltage', factor: 'acVoltage'},
                 {key: 'rmsVoltagePhB', name: 'voltage_phase_b', factor: 'acVoltage'},
                 {key: 'rmsVoltagePhC', name: 'voltage_phase_c', factor: 'acVoltage'},
+                {key: 'acFrequency', name: 'acfrequency', factor: 'acFrequency'},
```
Comment (Koenkk) [inline converters/fromZigbee.js:722]: `acfrequency` -> `ac_frequency`

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -733,6 +734,19 @@ const converters = {
             return payload;
         },
     },
+    powerfactor: {
+        cluster: 'haElectricalMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {};
+            if (msg.data.hasOwnProperty('powerFactor')) {
```
Comment (Koenkk) [inline converters/fromZigbee.js:742]: This logic can be added to the already existing `electrical_measurement` converter.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1171/files

File: converters/fromZigbee.js (hunk 1 above)
Comment (nukusinji) [reply converters/fromZigbee.js]: I fixed.

File: converters/fromZigbee.js (hunk 2 above)
Comment (nukusinji) [reply converters/fromZigbee.js:734]: I adding in electrical_measurement .

**Comment by Koenkk** [comment]
thanks!

**Pull request merged**